lib/tasks/etl_db_check.rake

```diff
@@ -0,0 +1,138 @@
+require 'net/http'
+require 'uri'
+
+namespace :etldb do
+    db_name = ENV['ETL_DB_NAME']
+    db_port = ENV['ETL_DB_PORT']
+    db_host = ENV['ETL_DB_HOST']
+    db_user = ENV['ETL_DB_USER']
+    db_pass = ENV['ETL_DB_PASSWORD']
+
+    tables = [
+        {
+            'title' => "Daily",
+            'name' => "daily_ohcl_prices",
+            'interval' => "''1 day''",
+            'url' => "https://hc-ping.com/772bacf5-958e-4541-b8ca-a81fe24c63d0"
+        },
+        {
+            'title' => "Hourly",
+            'name' => "hourly_ohcl_prices",
+            'interval' => "''1 hour''",
+            'url' => "https://hc-ping.com/c49dcf87-c6e5-4881-95f8-3bfad31d33ff"
+        }
+    ]
+
+    coins = [
+        {
+            'title' => "'Bitcoin",
+            'query' => lambda { Coin.where(symbol: 'BTC').pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "Ethereum",
+            'query' => lambda { Coin.where(symbol: 'ETH').pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "CoinFi",
+            'query' => lambda { Coin.where(symbol: 'COFI').pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "Top 10",
+            'query' => lambda { Coin.top(10).pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "Random 3",
+            'query' => lambda { Coin.where("ico_status=? OR ranking IS NOT NULL", 'listed')
```

Comment:
@anthotsang Actually this can probably just be `Coin.listed` since there's some metaprogramming in the `coin.rb` model that enabled this. The `ranking` is associated with CMC instead of CC which is where our data comes from so the `listed` scope is probably a better reflection of the data, apologies for the waffling!